Source/Forms/Gen4/Wild4.cpp

```diff
@@ -435,6 +426,16 @@ void Wild4::generatorEncounterIndexChanged(int index)
             break;
         }
 
+        if ((currentProfile->getVersion() & Game::HGSS) != Game::None && index > 2)
+        {
+            ui->toolButtonGeneratorLead->removeAction(tr("Suction Cups"));
+            ui->toolButtonGeneratorLead->addAction(tr("Suction Cups"), toInt(Lead::SuctionCups));
```

Comment:
Not the biggest fan of this approach. Should check if it exists and if not then add it. This check should be handled internally to the ComboMenu class

Reply:
done